server/utils.go

```diff
@@ -173,3 +176,127 @@ func fixGithubNotificationSubjectURL(url string) string {
 func fullNameFromOwnerAndRepo(owner, repo string) string {
 	return fmt.Sprintf("%s/%s", owner, repo)
 }
+
+// filter lines in a string from start to end
+func filterLines(s string, start, end int) (string, error) {
+	scanner := bufio.NewScanner(strings.NewReader(s))
+	var buf strings.Builder
+	i := 1
+	for scanner.Scan() {
```

Comment:
Any reason not to merge `i` into this line?
`for i := 1; i <= end || scanner.Scan(); i = i++`

Reply:
ohh I like it :+1: